Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -1,14 +1,39 @@
 import UIKit
+import CoreSpotlight
 
 /// A class used to display options for the note that is currently being edited
-class NoteOptionsViewController: UITableViewController {
+final class NoteOptionsViewController: UITableViewController {
 
     /// Array of `Section`s to display in the view.
     /// Each `Section` has `Rows` that are used for display
     fileprivate var sections: [Section] {
         return [optionsSection, linkSection, collaborationSection, trashSection]
     }
 
+    /// The note from the editor that we will change settings for
+    private var note: Note
```

Comment:
Would this property change at any time? (Same note for nearly all of the new properties!)

Reply:
A good spot, I've updated most to `let` now.